quinn-udp/src/unix.rs

```diff
@@ -337,7 +336,7 @@ fn recv(io: SockRef<'_>, bufs: &mut [IoSliceMut<'_>], meta: &mut [RecvMeta]) ->
                 hdrs.as_mut_ptr(),
                 bufs.len().min(BATCH_SIZE) as libc::c_uint,
                 0,
-                ptr::null_mut(),
+                std::ptr::null_mut(),
```

Comment:
Drive by review: is there a reason for changing this? Considering it may also need to be backported I'd reduce the changes to the minimum necessary to fix this

Reply:
There was a warning on macos after removing `ptr::read` from the other sections. If you're alright with the warning I can revert this line

Reply:
Can we put a cfg guard on the import matching the guard for this code?

Reply:
Won't this create another diff line that you are trying to avoid?

Reply:
Yeah, that's fair.